flink-ml-lib/src/test/java/org/apache/flink/ml/recommendation/SwingTest.java

```diff
@@ -140,6 +141,7 @@ public void testParam() {
         assertEquals(5, swing.getAlpha1());
         assertEquals(1, swing.getAlpha2());
         assertEquals(0.35, swing.getBeta(), 1e-9);
+        assertEquals(1, swing.getSeed());
```

Comment:
Let's also check the default value of `seed`.

Reply:
OK. The value of seed and different output while setting non-default value are tested in latest commit.